shared/src/userData.ts

```diff
@@ -68,6 +70,7 @@ export const createEmptyBusiness = (id?: string): Business => {
       dbaBusinessNameAvailability: undefined,
       lastVisitedPageIndex: 0,
     },
+    version: 0,
```

Comment:
pebble - This fn returns the shape of V147, wouldn't version 0 cause it to go through all the migrations? If we are keeping CURRENT_BUSINESS_VERSION = 1, shouldn't this be 1?

Reply:
This has been updated to the latest version.